test/resource.cpp

```diff
@@ -21,8 +21,8 @@
 using namespace camp::resources;
 
 // compatible but different resource for conversion test
-struct Host2 : Host {
-};
+//struct Host2 : Host {
```

Comment:
If you are going to leave this in and commented out, please add a comment why it is commented out.

Reply:
Well that I am still trying to work out. When Host2 is left in the test, MSVC and sycl compilers think there is an ambiguous operator== overload and it causes docker failures. When it's gone, everything passes. Not sure how to resolve this. I think Host2 should go back in so that the conversion tests can happen - but I need to figure out how to make MSVC and SYCL not complain about my operator== ...

Reply:
This was put back in and the problematic operator== functions removed